week-8/Homework/mandatory/2-fetch-exercise/exercise.js

```diff
@@ -17,10 +17,12 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch('*** Write the API address here ***')
+
+fetch('https://codeyourfuture.herokuapp.com/api/greetings')
     .then(function(response) {
         return response.text();
     })
     .then(function(greeting) {
-        // Write the code to display the greeting text here
+        let greetingDisplay = document.getElementById("greeting-text");
+        greetingDisplay.innerText = greeting
```

Comment:
Looks good! One small thing you could do is combine lines 26 and 27 to be `document.getElementById("greeting-text").innerText = greeting`

Reply:
You could also add error handling